include/oneapi/dpl/pstl/hetero/dpcpp/parallel_backend_sycl.h

```diff
@@ -1548,9 +1765,11 @@ struct __is_radix_sort_usable_for_type
 };
 
 #if _USE_RADIX_SORT
```

Comment:
I'm not sure if this and the following changes improve the readability.

Reply:
This is from `clang-format`. I will revert the changes for lines that I did not touch.